PxGraf.Frontend/src/api/services/table.ts

```diff
@@ -44,6 +44,15 @@ const fetchTable = async (idStack: string[], lang: string): Promise<ITableListRe
     return await client.getAsync(url, getParams);
 };
 
+export const sortTableData = (data: ITableListResponse[], primaryLanguage: string): ITableListResponse[] => {
```

Comment:
Maybe the api/services is not the best place for a utility sorting functions

Reply:
Makes sense. I'll move under src/utils in a separate file